### PR TITLE
Milestone 5: Styling

### PR DESCRIPTION
**Pull request by mdg0410**
> ## In this milestone:

✔ Add any missing JSX elements to your project (without adding extra functionality)
✔ Your final design should match Zeplin's design (this includes all elements + the same styling)
✔ You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/Styles/App.css [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-  padding: 1vh 5vh;
+  padding: 0;
+  background-color: rgb(247, 247, 247);
 }
 
-.wrapper {
+/* .wrapper {
   display: flex;
   flex-direction: column;
 }
```
Comment (zieeco) [inline src/Styles/App.css]: - Please, remove the commented piece of code to make your codebase clean.

File: src/Styles/BookItem.css [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  border: none;
+  background-color: #0290ff;
+  font-family: 'RobotoSlab Bold', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  cursor: pointer;
+}
```
Comment (zieeco) [inline src/Styles/BookItem.css:203]: - Let's use the correct font to make our design look close to the main design:

*main design*
image



*Your design*
image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @mdg0410,




Great job so far!  👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Styles/BookItem.css (hunk 2 above)
Comment (mdg0410) [reply src/Styles/BookItem.css:203]: > Correction 

## Import font-family: 
- Montserrat
- Roboto Slab

imagen

**Review by rotimiazeez — APPROVED** [review]
Hi @mdg0410 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good functioning app 👌🏽
- Great UI design 👍🏼
- Descriptive PR title and summary 👏🏾
- All linter checks passed ✅️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**